extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -410,4 +452,20 @@ private function formatTotalEditorsNumber( $editors ) {
 
 		return $editors;
 	}
+
+	/**
+	 * @param $userId
+	 * @return string name of DS svg file with proper badge
+	 */
+	private function getUserBadge( $userId ) {
+		$userGroups = User::newFromId( $userId )->getEffectiveGroups();
+
+		foreach ( self::PERMISSION_HIERARCHY as $group ) {
+			if ( in_array( $group, $userGroups ) ) {
+				return self::PERMISSIONS_TO_BADGES[ $group ];
+			}
+		}
+
+		return '';
```

Comment:
I think we should have some kind of default here. What does it mean if a user does not have any group rights?